examples/stress_tests/many_lights.rs

```diff
@@ -34,6 +34,9 @@ fn setup(
     mut meshes: ResMut<Assets<Mesh>>,
     mut materials: ResMut<Assets<StandardMaterial>>,
 ) {
+    #[cfg(debug_assertions)]
```

Comment:
It's important that we think about the fact that optimisation level and debug assertions are theoretically entirely orthogonal.

I know this is potentially contradictory to my suggestion to use `error`. However, in practise I do think that this is a fairly reasonable proxy.

Reply:
We could just always display the warning, no conditions: "this is a stress test, used to debug and benchmark the engine. it doesn't represent a real use case. be sure to run an optimised version"